src/Microsoft.TestPlatform.Extensions.BlameDataCollector/ProcessDumpUtility.cs

```diff
@@ -100,27 +99,38 @@ private static string BuildProcDumpArgs(int processId, string filename)
             // -accepteula: Auto accept end-user license agreement
             // -t: Write a dump when the process terminates.
             // This will create a minidump of the process with specified filename
-            return "-accepteula -t " + processId + " " + filename + ".dmp";
+            return "-accepteula -e 1 -g -t -f STACK_OVERFLOW -f ACCESS_VIOLATION " + processId + " " + filename + ".dmp";
```

Comment:
STACK_OVERFLOW and ACCESS_VIOLATION.. pick these values from a list. This list will contain only these 2 values for now but can be extended in future.

use a string builder to build the params.

Reply:
Done. Picking values from a list of exceptionFilters now.